nft-market/components/NFT/NFT.tsx

```diff
@@ -0,0 +1,159 @@
+                        <h3>{sale_conditions}</h3>
+                    </div>
+                    
+                }
+            </div>
+            <div className="flex mt-2 p-2 mx-2 bg-nh-purple-highlight rounded-xl">
+                <h1>{metadata.description}</h1>
+            </div>
+            <div className="flex flex-col justify-center p-2">
+                {sellOrBid ? 
+                    <>
+                        {!sell && <button className="bg-nh-purple hover:bg-nh-purple-highlight w-full p-2 text-black rounded-xl"
+                            onClick={async () => {
+                               setSell(true)
+                            }}
+                        >
+                            {translations.nft.sell}
+                        </button>}
+                        {sell &&
+                            <div className="flex flex-col gap-2">
+                                <input type="number" placeholder="Enter amount" 
+                                    className="p-2 text-center rounded-xl focus:outline-none bg-nh-purple-highlight text-white placeholder-white"
+                                    value={price} onChange={handlePrice}
+                                >
+                                </input>
+                                <div className="flex justify-center gap-2">
+                                    <button className="bg-nh-purple hover:bg-nh-purple-highlight p-2 w-full text-black rounded-xl"
+                                        onClick={async () => {
+                                            approveNFTForSale(token_id)
+                                        }}
+                                    >
+                                        {translations.nft.sell}
+                                    </button>
+                                    <button className="bg-nh-purple hover:bg-nh-purple-highlight p-2 w-full text-black rounded-xl"
+                                        onClick={async () => {
+                                            setSell(false)
+                                        }}
+                                    >
+                                        {translations.nft.cancel}
+                                    </button>
+                                </div>
+                            </div>
+                        
+                        }
+                    </>
+                    
+                :
+                <>
+                    {!bid && <button className={`bg-nh-purple hover:bg-nh-purple-highlight w-full p-2 text-black rounded-xl
+                        ${checkSellerIsOwner() ? "cursor-not-allowed" : ""}`}
+                        onClick={async () => {
+                            if(!checkSellerIsOwner()) {
+                                offerPrice(token_id, sale_conditions)
+                            } else {
+                                alert("Can't buy your own NFTs");
```

Comment:
Instead of alert try using popup or notifications dependency (used in previous example -> notifications folder)

Reply:
Added popup